src/devices_models/devices/DC_branches.jl

```diff
@@ -127,7 +127,7 @@ function branch_rate_constraints!(
     psi_container::PSIContainer,
     devices::IS.FlattenIteratorWrapper{B},
     model::DeviceModel{B, <:AbstractDCLineFormulation},
-    system_formulation::Union{
+    ::Union{
```

Comment:
This union is redundant (I know you didn't introduce it). 

since 

```julia 
PM.AbstractActivePowerModel <: PM.AbstractPowerModel
```

Reply:
Resolved in https://github.com/NREL-SIIP/PowerSimulations.jl/pull/560/commits/20ab12eff7f5adf899419a65ff2d2587a4cb9a95